client/post-editor/test/post-editor.jsx

```diff
@@ -1,91 +1,74 @@
+/** @jest-environment jsdom */
```

Comment:
All tests asset if spy injected in place of `tree.editor.setEditorContent` method was called or not. If it is called in the top level component then shallow rendering would simplify it a lot. I think it's a good candidate for further refactoring in the follow up PR.

/cc @aduth @nylen who know a bit more about the editor :)